.github/workflows/test_on_push.yaml

```diff
@@ -57,11 +95,17 @@ jobs:
           ./deps.sh
         env:
           CARTRIDGE_VERSION: ${{ matrix.cartridge-version }}
+          VSHARD_VERSION: ${{ matrix.vshard-version }}
 
-      - name: Install metrics
-        if: matrix.metrics-version != ''
+      # There is unable to install metrics with tt for Tarantool 2.11
```

Comment:
As example:

https://github.com/tarantool/crud/actions/runs/6416665739/job/17420846254

```
Installing http://rocks.tarantool.org/metrics-0.10.0-1.rockspec
Cloning into 'metrics'...
Note: switching to '391304b6e18ef3b0bca26c325c58fcd4dae9272b'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false


metrics 0.10.0-1 depends on lua >= 5.1 (5.1-1 provided by VM)
metrics 0.10.0-1 depends on checks >= 2.0.0 (3.3.0-1 installed)
metrics 0.10.0-1 is now installed in /home/runner/work/crud/crud/.rocks (license: BSD)

Checking stability of dependencies in the absence of
metrics scm-1...

Will not remove metrics scm-1.
Removing it would break dependencies for: 
cartridge-metrics-role 0.1.0-1

Use --force to force removal (warning: this may break modules).

Error: failed deploying files. The following files were not installed:
/home/runner/work/crud/crud/.rocks/share/tarantool/metrics/plugins/prometheus/init.lua
/home/runner/work/crud/crud/.rocks/share/tarantool/metrics/plugins/json/init.lua
/home/runner/work/crud/crud/.rocks/share/tarantool/metrics/plugins/graphite/init.lua
```

The `--force` flag does not help (but it helps locally).

Reply:
Actually this might be correct behavior:

https://github.com/tarantool/cartridge-metrics-role/blob/master/cartridge-metrics-role-scm-1.rockspec#L18

Reply:
It seems like incompatibility between cartridge (depends on cartridge-metrics-role since 2.8.0) and metrics. Don't see anything tt-related here

Reply:
Old metrics now tested with cartridge 2.7.9